cfpb/cfgov-refresh
# IA user testing mega menu changes behind feature flag

The IA team will be conducting user testing this week on the build server and need to test some changes to the consumer tools section of the mega menu. This PR adds a separate mega menu file with these changes & some associated css, hidden behind a feature flag.

## Additions

- Generated html for IA-testing mega menu in `mega-menu-for-ia-testing.html`
- Additional CSS for this version of mega menu in `mega-menu.less`, targeted using a class added to the menu in `mega-menu-for-ia-testing.html` 
- conditional in `base.html` that loads `mega-menu-for-ia-testing.html`  when the  `IA_USER_TESTING_MENU` flag is on

## Review
@schaferjh 
@cfpb/cfgov-backends 
@cfpb/cfgov-frontends 


## Testing

- Pull down branch
- Run gulp
- Navigate to http://localhost:8000/ and verify that the Consumer Tools section of the mega menu is unchanged
- Log in to the wagtail admin and add an IA_USER_TESTING_MENU feature flag
- Navigate to http://localhost:8000/ and verify that the Consumer Tools section of the mega menu has the same content/layout as the screenshot below

## Screenshots

<img width="1306" alt="screen shot 2016-12-12 at 9 51 42 am" src="https://cloud.githubusercontent.com/assets/778171/21110157/fc28c6fc-c050-11e6-868d-da4122444e67.png">

## Notes

Making these updates to the menu required significant structural changes to `_vars-mega-menu.html` and `mega-menu.html` that will probably involve some discussion. For the purposes of this test, it seemed simpler to add a single file with the generated html for the menu. Once the tests are over, the code behind the feature flag can be removed and another PR made if/when any of these changes are made permanent. I can upload the branch with the structural changes and/or swap out those files here if that's preferred, though.

## Checklist
